.env.default

```diff
@@ -15,4 +15,5 @@ export TEST_DATABASE_USER=
 export TEST_DATABASE_USER_PASSWORD=
 
 
-export MW_FEATURE_BRAINSTORMING_TEASER=true
\ No newline at end of file
+export MW_FEATURE_BRAINSTORMING_TEASER=true
+export MW_DEFAULT_LOCALE=en
```

Comment:
maybe add a short comment, that you need either to recompile the app or just run `mix gettext.extract` to switch the language after a successful build